DEVELOP.md

```diff
@@ -0,0 +1,43 @@
+# Development Guide
+
+[![Build status](https://badge.buildkite.com/ce01baf77e07a728f3d80575254634c3d63d8a5eda69ba7fb3.svg?branch=master)](https://buildkite.com/operable/cog)
+
+## Local Setup
+
+Cog depends on a few things to run locally:
+
+    * Postgres
+    * A `SLACK_API_TOKEN` environment variable (with a valid token)
+
```

Comment:
You may want to say that a chat adapter application with the necessary environment variables are set, such as Slack, HipChat, IRC. Then point them to the config file to determine the necessary variables, stating that the 'SLACK_API_TOKEN' must be set when using Cog out of the box.